### PR TITLE
feat: implement Token-based test authentication

### PR DESCRIPTION
**Pull request by TerryW-tx**
Implement the JWT feature for example. I'm not sure whether it will work since I'm not clear how the test cases will realize their authentication. If this code is a feasible implementation, I'll add other authentication functions later. Please give me some advice, thx.

mod: since this auth feature is not only feasible to JWT, rename it to token authentication.

### CODE REVIEW

File: ytest/auth.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ytest
+
+import "net/http"
+
+type JwtAuth struct {
```
Comment (xushiwei) [inline ytest/auth.go:21]: don't public this class

File: ytest/auth.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	// Type     string
+	JwtToken string
+}
+
+type JwtRoundTripper struct {
```
Comment (xushiwei) [inline ytest/auth.go:26]: don't public this class

File: ytest/auth.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	RoundTripper http.RoundTripper
+	Token        string
+}
+
+func (rt *JwtRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	req.Header.Set("Authorization", rt.Token)
```
Comment (xushiwei) [inline ytest/auth.go:32]: according implementation, there is nothing about `jwt`.
so please change to another class name.

Comment (TerryW-tx) [reply ytest/auth.go:32]: Thanks for your comments. JwtAuth is already renamed to tokenAuth.

**Pull request merged**